tests/acceptance/features/webUITrashbin/trashbinFilesFolders.feature

```diff
@@ -10,37 +10,37 @@ Feature: files and folders exist in the trashbin after being deleted
     And the user has browsed to the files page
 
   @smokeTest
-  @skip @yetToImplement
   Scenario: Delete files & folders one by one and check that they are all in the trashbin
     When the user deletes the following elements using the webUI
       | name                                  |
       | simple-folder                         |
       | lorem.txt                             |
       | strängé नेपाली folder                 |
       | strängé filename (duplicate #2 &).txt |
+    And the user browses to the trashbin page using the webUI
     Then as "user1" folder "simple-folder" should exist in trash
     And as "user1" file "lorem.txt" should exist in trash
     And as "user1" folder "strängé नेपाली folder" should exist in trash
     And as "user1" file "strängé filename (duplicate #2 &).txt" should exist in trash
-    And the deleted elements should be listed in the trashbin on the webUI
-    And file "lorem.txt" should be listed in the trashbin folder "simple-folder" on the webUI
+    And the deleted elements should be listed on the webUI
+    And file "lorem.txt" should be listed in the folder "simple-folder" on the webUI
 
-  @skip @yetToImplement
   Scenario: Delete a file with problematic characters and check it is in the trashbin
-    When the user renames the following file using the webUI
+    When the user renames the following files using the webUI
```

Comment:
this is preparation, make it an API Given step or use files with difficult names that are already in the skeleton folder